### PR TITLE
fix: refactored code

### PR DESCRIPTION
**Pull request by RahulSingh9131**

### CODE REVIEW

**Review by rohit8686 — APPROVED** [review]
- Code looks clean 
- Good to merge

**Review by shuntzhere — APPROVED** [review]
Everything looks good and seems to be working fine, go ahead with the merge.

File: components/BookmarkPost.js [hunk 1]
```diff
@@ -3,14 +3,15 @@ import Moment from 'react-moment';
...
+import Image from 'next/image';
 
 function BookmarkPost({_id,post,postPage}) {
     const dispatch=useDispatch();
 
   return (
     <div className='p-3 flex cursor-pointer border-b border-gray-700'>
         {!postPage && (
-            <img 
+            <Image
```
Comment (vaibhav-mahalle) [inline components/BookmarkPost.js:14]: you could have named the component little better I feel.

**Review by vaibhav-mahalle — COMMENTED** [review]
Please add a preview link or screenshot in your pr, suggesting some minor changes. Go ahead and merge it

**Pull request merged**